perdido/app.js

```diff
@@ -0,0 +1,221 @@
+(function() {
```

Comment:
I intentionally separate the setup and styles to encourage users to do the same, see https://github.com/jsstyles/examples/blob/gh-pages/calendar/components/jss.js